spec/requests/api/merge_requests_spec.rb

```diff
@@ -34,7 +37,7 @@
         expect(response.status).to eq(200)
         expect(json_response).to be_an Array
         expect(json_response.length).to eq(3)
-        expect(json_response.last['title']).to eq(merge_request.title)
+        expect(json_response.last["title"]).to eq(merge_request.title)
```

Comment:
Prefer single-quoted strings when you don't need string interpolation or special symbols.